h/sql_tasks/tasks/report/create_from_scratch/00_schema/01_create.jinja2.sql

```diff
@@ -0,0 +1,2 @@
+DROP SCHEMA IF EXISTS report CASCADE;
+CREATE SCHEMA report AUTHORIZATION {{db_user}};
```

Comment:
Oh no idea. A lot of this could be cargo cult type stuff. Honestly if something like this worked I've probably moved on. It does seem better to limit it to the user in question if we can?